docs/migration-guides/1.1-1.2.md

```diff
@@ -0,0 +1,29 @@
+# Migrate from v1.1 to v1.2
+
+## Event Bus
+
+The 1.2 release updates the Knative eventing version to 0.5 from 0.4. This brings in considerable changes to the Knative eventing CRDs which cannot be automatically done in Kyma update. To accommodate them, the following post update steps needs to be followed.
+
+1. Delete all the existing Knative eventing subscription custom resources. They will be recreated after all  the steps are completed.
+
+    ```bash
+    kubectl delete subscriptions.eventing.knative.dev --all -n kyma-system
+    ```
+ 
+2. Delete all the existing Knative eventing channels custom resources. They will be recreated after all  the steps are completed.
+    
+    ```bash
+    kubectl delete channels.eventing.knative.dev --all -n kyma-system
+    ```
+
+3. Apply all the Knative eventing CRDs that have delta in 0.5 release.
```

Comment:
```suggestion
3. Apply all the Knative eventing CRDs that have delta in 0.5 release.
```
One question: what does `delta` mean?  As far as I know, it means kind of comparison - so do we here mean the parts valid for 0.4 and 0.5?

Reply:
oh, by delta I meant any changes in the CRDs like some new fields or definitions

Reply:
Ok, reworked this part :)